api/src/main/proto/spine/protodata/ast.proto

```diff
@@ -74,7 +74,7 @@ message TypeName {
     //
     // Must not include a trailing slash ("/").
     //
-    string type_url_prefix = 4 [(pattern).regex = "^.*[^/]$"];
+    string type_url_prefix = 4 [(pattern).regex = "^.*[^\\/]$"];
```

Comment:
I am still not sure whether `\\` is what we need. Does it work properly without the backslashes?

Reply:
>> Regarding (5): Java Regex treats \ as en escape for /, which indeed does not even have to be escaped. So, [^\\/] is the same as [^/].

https://github.com/SpineEventEngine/validation/pull/195#issuecomment-2700946186

Reply:
They make no difference.